service/controller/chart/resource/release/update.go

```diff
@@ -111,12 +111,21 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange inte
 			return nil
 		}
 
-		if helmclient.IsInvalidManifest(err) {
+		if helmclient.IsValidationFailedError(err) {
 			reason := err.Error()
 			reason = fmt.Sprintf("helm validation error: (%s)", reason)
 			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("helm release %#q failed, %s", releaseState.Name, reason))
 			addStatusToContext(cc, reason, validationFailedStatus)
 
+			r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
+			resourcecanceledcontext.SetCanceled(ctx)
+			return nil
+		} else if helmclient.IsInvalidManifest(err) {
+			reason := err.Error()
+			reason = fmt.Sprintf("Invalid manifest error: (%s)", reason)
```

Comment:
```suggestion
			reason = fmt.Sprintf("invalid manifest error: (%s)", reason)
```

Can we make this lower case for consistency with the other messages?

Reply:
Thx, I fixed it for both create/update.go